code/modules/reagents/chemistry/holder.dm

```diff
@@ -495,6 +495,9 @@ var/const/INJECT = 5 //injection
 /datum/reagents/proc/add_reagent(reagent, amount, list/data=null, reagtemp = 300, no_react = 0)
 	if(!isnum(amount) || !amount)
 		return 1
+	if(!isnum(reagtemp) || !reagtemp)
+		message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[usr.x];Y=[usr.y];Z=[usr.z]'>(JMP)</a>.")
```

Comment:
message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[usr.x];Y=[usr.y];Z=[usr.z]'>(JMP)</a>.")

should be

message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[my_atom.x];Y=[my_atom.y];Z=[my_atom.z]'>(JMP)</a>.")

(i am shit at github specifically)